bukkit/src/main/java/hu/montlikadani/tablist/bukkit/tablist/groups/Groups.java

```diff
@@ -131,14 +139,25 @@ public void load() {
 	}
 
 	/**
-	 * Sets the player prefix, suffix and tab name except if the player is hidden.
+	 * Sets the player prefix, suffix, tab name, and position
+	 * on tablist, except if the player is hidden.
 	 * 
 	 * @param groupPlayer {@link GroupPlayer}
-	 * @param priority
+	 * @param safePriority Safe priority value. Should be between 0 and 999999999.
 	 */
-	public void setPlayerTeam(GroupPlayer groupPlayer, int priority) {
-		if (groupPlayer != null && !groupPlayer.getUser().isHidden()) {
-			groupPlayer.getTabTeam().setTeam(groupPlayer.getFullGroupTeamName());
+	public void setPlayerTeam(GroupPlayer groupPlayer, int safePriority) {
+		if (safePriority < 0) {
+			safePriority = 0;
+		}
+
+		if (safePriority > 999999999) {
+			safePriority = 999999999;
```

Comment:
`Integer.MAX_VALUE`

Reply:
safePriority must have a maximum value of 999999999